int16[] id
geometry_msgs/Vector3[] position
geometry_msgs/Vector3[] velocity